cartridges/cloudinary_sfra_changes/cartridge/templates/default/product/components/productTileImage.isml

```diff
@@ -2,8 +2,9 @@
     <a href="${pdict.urls.product}">
         <iscomment>Custom start: add cloudinary product image</iscomment>
         <isif condition="${product.CLDEnabled && product.images && product.images.cloudinaryProductImage}">
-            <isset name="isResponsive" value="${product.images.cloudinaryProductImage.isResponsive}" scope="page" />
-            <img class="tile-image cld-responsive" 
+            <isset name="isResponsive" value="${!product.images.cloudinaryProductImage.isResponsive}" scope="page" />
```

Comment:
why was the negation added (!product.images.cloudinaryProductImage.isResponsive) ?